---
output: 
    md_document:
        variant: markdown_github
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```

<style type="text/css">
.table {
    width: 40%;
}
</style>

## Example data formats for the sDIV project *sEcoEvo: Biodiversity Dynamics â€“ The Nexus Between Space & Time*

We are providing example data for all the different data types we are interested in gathering for communities.

* Time calibrated trees (with branch lengths even if unresolved)
  
    > An example tree in Newick format is in the file **example.newick**. Please note that trees can include both sampled and unsampled lineages (the example tree includes 7 lineages that were not sampled in the local community). The structure of the tree looks like this:


```{r}
knitr::include_graphics('https://github.com/continuousity/example_data/raw/master/example_tree.jpeg', 
                        auto_pdf = FALSE, dpi = NA)
```

* Abundances 

    > We provide an example site-by-species file including 3 sites and the 5 sampled species (B, D, E, F, & K) in the file **example_abundances.txt**. This is a simple tab-delimited text file (but can also be a CSV file) with rows corresponding to sites, and columns corresponding to species, cells contain the abundance of each species at each site.

```{r}
x <- read.table('https://raw.githubusercontent.com/continuousity/example_data/master/example_abundances.txt', header = TRUE)
knitr::kable(x, caption = '')
```

* Per location per taxon sequence data (including identical sequences). 

    > Example sequence data is shown in the **fastqs/** directory, with one fastq file per site, including all sequences for all individuals sequenced at that site. We will use these sequences to calculate nucleotide diversity per species per site.

* Sample/species/site mapping file

    > Sequence data need to be matched to species and sites. We include an example file called **example_pops.txt** which does this. This is a tab delimited text file (but could also be CSV) where each row contains 3 items, the sample id (which should match the sample name in the fastq files), the species ID (which should match the ID in the newick tree), and the sample site (which should match the site name in the abundances file).

```{r}
seqMap <- read.table('https://raw.githubusercontent.com/continuousity/example_data/master/example_pops.txt', header = TRUE)
knitr::kable(head(seqMap), caption = '')
```

* Site metadata file

    > We need to know a few things about sites, most importantly their goegraphic locations, and hopefully also something about sampling methodology such as area, effort, sampling technique.  The included example file **site_metadata.txt** (tab delimited in this case, but CSV is equally good) refers to arthropod pitfall trapping as an example

```{r}
siteMap <- read.table('https://raw.githubusercontent.com/continuousity/example_data/master/site_metadata.txt', header = TRUE, sep = '\t')
knitr::kable(siteMap, caption = '')
```
